azure-pipelines/mirror-branch.yaml

```diff
@@ -0,0 +1,17 @@
+trigger:
+- main
+pr: none
+jobs:
+  - job: mirror
+    steps:
+    - task: TriggerBuild@4
```

Comment:
can you add some flavor / comments linking back to things like https://github.com/Azure/pipelines/issues/101, so future folks are aware of the design choices here?

Reply:
This has been difficult to test locally, I'd like to get this file added to the repo so I can experiment with it a bit and see if this approach will work.